pkg/controller/gc/gc_controller.go

```diff
@@ -77,6 +77,8 @@ func New(kubeClient clientset.Interface, resyncPeriod controller.ResyncPeriodFun
 		&api.Pod{},
 		resyncPeriod(),
 		framework.ResourceEventHandlerFuncs{},
+		// We don't need to build a index for podStore here
+		cache.Indexers{},
```

Comment:
For consistency, can you add `cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc}`?